app/views/job_runs/index.html.erb

```diff
@@ -15,15 +15,7 @@
       </tr>
     </thead>
     <tbody>
-    <% @job_runs.each do |job_run| %>
-      <tr>
-        <td><%=link_to "Job ##{job_run.id}",job_run%></td>
```

Comment:
Did you notice that the title of the link here is different?

Reply:
yeah, noticed that and fixed it by missing in a local variable (which I think requires the 'partial' declaration?

Reply:
You can just pass a hash of locals after the partial name as keyword args.